apps/dashboard/src/pages/test-workflow.tsx

```diff
@@ -18,7 +18,7 @@ export const TestWorkflowPage = () => {
     <>
       <PageMeta title={`Trigger ${workflow?.name}`} />
       <EditWorkflowLayout headerStartItems={<EditorBreadcrumbs />}>
-        {testData && <TestWorkflowTabs testData={testData} />}
+        <TestWorkflowTabs testData={testData} />
```

Comment:
This was causing a flicker when switching tabs which didn't looked good, now the transition is smoother